<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCR_test_Dialog</class>
 <widget class="QDialog" name="DCR_test_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCR测试界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/img.qrc">
    <normaloff>:/new/prefix1/myappico.ico</normaloff>:/new/prefix1/myappico.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,10,1">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>DCR 功能测试</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTableWidget" name="tableWidget_4">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>166</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="tableWidget_5">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTableWidget" name="tableWidget_6">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QTableWidget" name="tableWidget_7">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTableWidget" name="tableWidget_8">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>166</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QTableWidget" name="tableWidget_9">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTableWidget" name="tableWidget_10">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QTableWidget" name="tableWidget_11">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QTableWidget" name="tableWidget_12">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>166</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QTableWidget" name="tableWidget_13">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTableWidget" name="tableWidget_14">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QTableWidget" name="tableWidget_15">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QTableWidget" name="tableWidget_16">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>65</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QTableWidget" name="tableWidget_17">
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>45</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1,1,1,1,1">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>558</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>VsPad电压(0x)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="votage_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DCR时长：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="timeOffset_comboBox">
       <item>
        <property name="text">
         <string>00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>06</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>07</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>08</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>09</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0E</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>开始采集</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/img.qrc"/>
 </resources>
 <connections/>
</ui>
